optimise

bruno-unna opened the pull request:
I've removed some critical parts of the system. Could use a review.

Commit:
Delete the useless method `nothing`

Commit:
Enrich the description of the project

qiuyangnie commented on src/main/scala/com/getvisibility/learning/lists/Lists.scala:
```diff
@@ -32,7 +32,4 @@ object Lists {
-
-  /** This method does what its name says */
-  def nothing(input: Int): Int = 0
 }
```
I like this change!

gayatri-bhadane replied:
Its fine.

qiuyangnie commented on src/test/scala/com/getvisibility/learning/lists/ListsTest.scala:
```diff
@@ -28,4 +28,6 @@ class ListsTest extends AnyWordSpec {
   }
+
+  // TODO please check that all methods have unit tests
 }
```
Remove everything!

qiuyangnie reviewed (approved):
Looks good

qiuyangnie commented:
I think it is okay, but the changes are so critical that we should wait for @gayatri-bhadane to review

gayatri-bhadane commented on src/test/scala/com/getvisibility/learning/lists/ListsTest.scala:
```diff
@@ -28,4 +28,6 @@ class ListsTest extends AnyWordSpec {
     }
   }
+
+  // TODO please check that all methods have unit tests
```
Unwanted comment, so we can remove it.

bruno-unna replied:
Will do. But I'm very fond of my comments, and you've broken my heart.

gayatri-bhadane reviewed (changes requested):
Code is ok but you need to update your comments.

Commit:
Address Gayatri's review

She was unhappy about the TODO comment in `ListsTest`, so instead of
fixing the test, I've removed the comment.

bruno-unna commented on src/test/scala/com/getvisibility/learning/lists/ListsTest.scala:
```diff
@@ -28,6 +28,4 @@ class ListsTest extends AnyWordSpec {
   }
-
-  // TODO please check that all methods have unit tests
 }
```
Done. Happy now?
